CMakeLists.txt

```diff
@@ -37,11 +41,35 @@ set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
 set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreadedDLL)
 set(CMAKE_POSITION_INDEPENDENT_CODE ${BUILD_SHARED_LIBS})
 
+# Toolchain vended dependencies
 find_package(dispatch CONFIG)
 find_package(Foundation CONFIG)
 
-find_package(SwiftASN1 CONFIG REQUIRED)
-find_package(SwiftCrypto CONFIG REQUIRED)
+include(FetchContent)
+
+set(_CERT_SAVED_BUILD_EXAMPLES ${BUILD_EXAMPLES})
+set(_CERT_SAVED_BUILD_TESTING ${BUILD_TESTING})
+
+find_package(SwiftASN1 CONFIG)
+if(NOT SwiftASN1_FOUND)
+  message("-- Vending swift-asn1")
+  FetchContent_Declare(ASN1
+    GIT_REPOSITORY https://github.com/apple/swift-asn1
+    GIT_TAG 1.1.0)
+  FetchContent_MakeAvailable(ASN1)
+endif()
+
+find_package(SwiftCrypto CONFIG)
+if(NOT SwiftCrypto_FOUND)
+  message("-- Vending swift-crypto")
+  FetchContent_Declare(Crypto
+    GIT_REPOSITORY https://github.com/apple/swift-crypto
+    GIT_TAG 3.0.0)
```

Comment:
```suggestion
    GIT_TAG 3.2.0)
```

Reply:
Do we want these versions to match update-checkout or it doesn't matter?

Reply:
Lets go ahead and match it :)

Reply:
The problem is update-checkout use swift-asn1 1.0.0 and swift-crypto 3.0.0. lol

But if it's ok to use the latest (i.e., 1.1.0 and 3.2.0) then let's leave it.